sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -362,7 +360,8 @@ public String toString() {
      * generic create a sub-type of {@link TypeReference}, if the type is non-generic use {@link
      * TypeReference#createInstance(Class)}.
      * <p>
-     * <b>Note:</b> A {@link JsonSerializer} implementation must be available on the classpath.
+     * <b>Note:</b> This method first looks for a {@link JsonSerializer} implementation on the classpath. If no
```

Comment:
We load `JsonSerializerProvider` to create a `JsonSerializer` instead of looking for a `JsonSerializer` directly.
Users will think that an json serializer implementation will be loaded automatically if there is one on the classpath. But this isn't accurate.

Reply:
Updated javadoc